api-guidelines/v4.0/API-DESIGN-GUIDELINES/ED-FI-DESCRIPTORS.md

```diff
@@ -1,29 +1,33 @@
 # Ed-Fi Descriptors
 
 Descriptors in the Ed-Fi Data Standard are a set of mechanisms to support
-flexible enumerations or code tables. Each Descriptor has the following
-attributes:
+flexible enumerations or code sets. The following table defined the attributes
+of an Ed-Fi Descriptor:
 
-* namespace
-* codeValue
-* shortDescription
-* description
-* effectiveBeginDate
-* effectiveEndDate
+**Table 3.** Descriptor Attributes
 
-The GET of a resource _must_ return the namespace and codeValue for Descriptor
-enumerations. Other components of the Descriptor can be retrieved from the
-Descriptor resource.
+The `GET`, `POST`, and `PUT` columns indicate if the attribute is _required_ ("yes"), _optional_ ("opt"), or _must not have_ ("no")k.
 
-The PUT or POST of a resource _must_ specify the namespace and codeValue for
-each Descriptor value in a descriptor reference (described below).
+| Attribute              | `GET` | `POST` | `PUT` | Notes                                                                                  |
+| ---------------------- | ----- | ------ | ----- | -------------------------------------------------------------------------------------- |
+| `namespace`            | yes   | yes    | yes   |                                                                                        |
+| `codeValue`            | yes   | yes    | yes   | Value _preferred_ to be one or more words rather than a numeric code or random string. |
+| `shortDescription`     | yes   | yes    | yes   |                                                                                        |
+| `description`          | yes   | opt    | opt   | Longer description that may contain additional normative usage guidance.               |
+| `effectiveBeginDate`   | yes   | opt    | opt   | Date for display only, not validation                                                  |
+| `effectiveEndDate`     | yes   | opt    | opt   | Date for display only, not validation                                                  |
+| `id`                   | yes   | no     | yes   |                                                                                        |
+| `_etag`                | yes   | no     | no    |                                                                                        |
```

Comment:
Etag is optional -- probably need some kind of asterisk statement about what's expected if the implementation doesn't do etags